diff --git a/app/build.gradle b/app/build.gradle
index 8c7031d..678bd4d 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -10,5 +10,5 @@ application {
     }
 }
 
-//tasks.withType(AbstractLinkTask.class).configureEach { asNeeded.enable() }
-tasks.withType(AbstractLinkTask.class).configureEach { linkerArgs.add('-Wl,--as-needed') }
+tasks.withType(AbstractLinkTask.class).configureEach { asNeeded.enable() }
+//tasks.withType(AbstractLinkTask.class).configureEach { linkerArgs.add('-Wl,--as-needed') }
